docker/dev/docker-compose.yml

```diff
@@ -0,0 +1,33 @@
+#  Copyright 2021 Collate
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#  http://www.apache.org/licenses/LICENSE-2.0
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+
+version: "3.9"
+services:
+  mysql:
```

Comment:
@amiorin please make sure you create separate PRs for each issue. This way it will be easier to review and not cause confusion when we are merging in.

Reply:
Will do. This one is a catch-all PR Draft that I was planning to split later.